cns/restserver/restserver.go

```diff
@@ -67,6 +68,8 @@ type HTTPRestService struct {
 	dncPartitionKey    string
 	EndpointState      map[string]*EndpointInfo // key : container id
 	EndpointStateStore store.KeyValueStore
+	supportedApisCache []string
```

Comment:
the name doesn;t indicate that this is supported APIs from NMA. Need the name to reflect that